### PR TITLE
Initialize the simd type when initialize an indexnode

### PR DESCRIPTION
**Pull request by longjiquan**
Signed-off-by: dragondriver <jiquan.long@zilliz.com>
issue: #7963 #7954

### CODE REVIEW

File: internal/indexnode/indexnode.go [hunk 1]
```diff
@@ -90,6 +102,15 @@ func (i *IndexNode) Register() error {
 	return nil
 }
 
+func (i *IndexNode) initKnowhere() {
```
Comment (cydrain) [inline internal/indexnode/indexnode.go:105]: indexnode should not call SegcoreInit, instead you should add IndexBuilderInit and call it.

Comment (DragonDriver) [reply internal/indexnode/indexnode.go:105]: OK, get it.

File: internal/indexnode/param_table.go [hunk 2]
```diff
@@ -58,8 +62,12 @@ func (pt *ParamTable) InitAlias(alias string) {
 func (pt *ParamTable) Init() {
 	once.Do(func() {
 		pt.BaseTable.Init()
+		if err := pt.LoadYaml("advanced/knowhere.yaml"); err != nil {
```
Comment (xiaocai2333) [inline internal/indexnode/param_table.go:65]: I think the yaml file should be loaded in BaseTable.

Comment (DragonDriver) [reply internal/indexnode/param_table.go:65]: this config file can only be loaded into querynode and indexnode. so it's ok to be loaded in need.

**Pull request merged**